mule-migration-tool-e2e-tests/src/test/resources/e2e/oauth2/tweetbook-oauth2-provider/output/src/main/resources/migration/HttpListener.dwl

```diff
@@ -0,0 +1,29 @@
+%dw 2.0
+
+/**
+ * Emulates the response headers building logic of the Mule 3.x HTTP Connector.
+ */
+fun httpListenerResponseHeaders(vars: {}) = do {
```

Comment:
I think we are not comparing these files in other tests, aren't we? Do we want to add them to all the tests? or remove from this one to keep consistency?

Reply:
I've added them in order to have at lease one test that verifies that those files are generated, somewhat like the html report test, but maybe it should be a separate test for that purpose only